### PR TITLE
About page (UI)

### LINKED ISSUE #11: About Page UI

### PR DESCRIPTION
**Pull request by avnogy**
resolves #11

### CODE REVIEW

File: cv_next/app/login/page.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                    height={6}
+                  />
+                  <span>Login with Google</span>
+                </button>
+              </div>
+              <div>
+                <button className="flex items-center justify-center gap-x-2 rounded-md border border-primary bg-accent px-4 py-3 text-primary transition hover:text-muted">
+                  <Image
+                    className="h-6 w-6"
+                    src="something.svg"
```
Comment (ShakedAp) [inline cv_next/app/login/page.tsx:27]: is this right?

**Review by ShakedAp — APPROVED** [review]
Looks ok!

File: cv_next/app/login/page.tsx (hunk 1 above)
Comment (ShakedAp) [reply cv_next/app/login/page.tsx:27]: hellothere

**Comment by Aradon267** [comment]
Moved to #76

**Pull request closed without merging**